src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicencePolicyServiceTest.kt

```diff
@@ -17,4 +23,85 @@ class LicencePolicyServiceTest() {
   fun `All versions are accessible`() {
     assertThat(licencePolicyService.allPolicies()).hasSize(3)
   }
+
+  @Test
+  fun `No policy versions have duplicate condition codes within the same policy`() {
+    licencePolicyService.allPolicies().forEach { policy ->
+      assertThat(policy.allAdditionalConditions().map { it.code }).doesNotHaveDuplicates()
+      assertThat(policy.allStandardConditions().map { it.code }).doesNotHaveDuplicates()
+    }
```

Comment:
check that we don't every use the same id more than twice within a set of conditions